object.go

```diff
@@ -15,6 +15,11 @@ type PyObject struct {
 	ptr *C.PyObject
 }
 
+// String returns a string representation of the PyObject
+func (self *PyObject) String() string {
+	return PyString_AsString(self.Str())
```

Comment:
`self.Str()` returns a new reference.
I believe this should read as:
```go
func (self *PyObject) String() string {
	o := self.Str()
	defer o.DecRef()

	return PyString_AsString(o)
}
```

Reply:
updated, good catch.